From 05a8a48ea58534bdecfe714383348a7c120f5679 Mon Sep 17 00:00:00 2001
From: "Tobias C. Berner" <tcberner@FreeBSD.org>
Date: Wed, 1 Aug 2018 13:38:16 +0200
Subject: [PATCH] Fix build on FreeBSD by including <array>.

latte-dock-0.8.0/app/dock/dockview.h:367:44:
    error: implicit instantiation of undefined template 'std::__1::array<QMetaObject::Connection, 4>'
    std::array<QMetaObject::Connection, 4> connectionsManagedLayout;
                                           ^
---
 app/dock/dockview.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git app/dock/dockview.h app/dock/dockview.h
index 7c4815f5..9f608063 100644
--- app/dock/dockview.h
+++ app/dock/dockview.h
@@ -28,6 +28,8 @@
 #include "../plasmaquick/configview.h"
 #include "../../liblattedock/dock.h"
 
+#include <array>
+
 #include <QQuickView>
 #include <QQmlListProperty>
 #include <QMenu>
-- 
2.18.0

